server/libs/infra/src/search/typesense.repository.ts

```diff
@@ -238,6 +238,22 @@ export class TypesenseRepository implements ISearchRepository {
     throw new Error(`Invalid collection: ${collection}`);
   }
 
+  async vectorSearch(input: number[]): Promise<SearchResult<AssetEntity>> {
+    const alias = await this.client.aliases(SearchCollection.ASSETS).retrieve();
+
+    const { results } = await this.client.multiSearch.perform({
+      searches: [
+        {
+          collection: alias.collection_name,
+          q: '*',
+          vector_query: `smartInfo.clip:([${input.join(',')}], k:100)`,
+        } as any,
+      ],
+    });
```

Comment:
I think this search currently doesn't scope by userId, is that right?

Reply:
Ah, yeah forgot to add that back in

Reply:
Also, from playing around on the preview a little: It seems like we're always returning 10 results exactly, and not bounding by a minimum score, is that correct?

Reply:
Great question, will have to research what the default page size is here

Reply:
Scoped by user id, an no idea what minimum distance for cutting off results, limited to 100 results though.